main.go

```diff
@@ -119,6 +133,19 @@ func getNamespace() (string, error) {
 	return ns, nil
 }
 
+// The name of the secret containing the addon params
+func getEnvVars(envVars *EnvVars) error {
```

Comment:
Would prefer to have this method read all env vars, this includes the namespace.